tests/ui/test_ui.py

```diff
@@ -217,9 +217,21 @@ def test_keypress_ALL_MENTIONS(self, view, mocker, key):
         view.model.controller.show_all_mentions.assert_called_once_with(view)
         assert view.body.focus_col == 1
 
-    @pytest.mark.parametrize('autohide', [True, False], ids=[
-        'autohide', 'no_autohide'])
-    def test_keypress_w(self, view, mocker, autohide):
+    def ids_autohide(param):
+        if isinstance(param, bool):
+            return 'autohide' if param else 'no_autohide'
+
+    @pytest.mark.parametrize('autohide, key', [
+            (autohide, key)
+            for autohide, keys in [
+                (True, keys_for_command('SEARCH_PEOPLE')),
+                (False, keys_for_command('SEARCH_PEOPLE')),
```

Comment:
Given the duplication, could we just loop this over the bool values? Same below.